worlds/alttp/docs/multiworld_fr.md

```diff
@@ -1,39 +1,25 @@
 # Guide d'installation du MultiWorld de A Link to the Past Randomizer
 
-<div id="tutorial-video-container">
-    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/mJKEHaiyR_Y" frameborder="0"
-      allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
-    </iframe>
-</div>
-
 ## Logiciels requis
 
-- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases)
-- [QUsb2Snes](https://github.com/Skarsnik/QUsb2snes/releases) (Inclus dans les utilitaires précédents)
+- [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases). N'oubliez pas de cocher `SNI Client - A Link to the Past Patch Setup` lors de l'installation.
+- [SNI](https://github.com/alttpo/sni/releases). Inclus avec l'installation d'Archipelago ci-dessus.
+- SNI n'est pas compatible avec (Q)Usb2Snes.
```

Comment:
This point should be indented more. It's not a required software but a precision on SNI

Reply:
Would also need to be indented on the english version